main.go

```diff
@@ -28,8 +28,8 @@ var (
 	analysisWorkers = flag.Int("analysisworkers", 32, "number of analysis workers")
 	profiles        = flag.StringSlice("profile", []string{}, "profile types to store (one or more of cpu, heap, block)")
 
-	filter     = flag.StringP("filter", "f", "", "regex pattern of cache keys to track")
-	reportSize = flag.IntP("top", "t", 100, "number of keys to report")
+	filter     = flag.String("filter", "", "regex pattern of cache keys to track")
+	format     = flag.StringP("format", "f", "key,max(size),sum(size)", "data and aggregates to display")
```

Comment:
document in --help display the available fields (key, size) and aggregator operators (max, sum, count)